app/commands/put_publish_intent.rb

```diff
@@ -5,7 +5,15 @@ def call
 
       if downstream
         payload = publish_intent
-        Adapters::ContentStore.put_publish_intent(base_path, payload)
+        enqueue = ENV.fetch("ENQUEUE_PUBLISH_INTENTS", false)
+        if enqueue == "true"
```

Comment:
is there a reason this is stored as a string rather than a boolean?

Reply:
Yes - it's passed through as an environment variable from the Kubernetes Helm chart, which means it's ultimately going to passed through to the app as a string value:

```
$ RANDOM_ENV_VAR=true irb
irb(main):001> ENV['RANDOM_ENV_VAR']
=> "true"
```

Reply:
totally agree